### PR TITLE
[SW-1022] Decide Explorer for Bitcoin & Dogecoin

### PR DESCRIPTION
**Pull request by pstachel**
…

### CODE REVIEW

File: src/popup/components/AddressTruncated.vue [hunk 1]
```diff
@@ -22,12 +22,17 @@
...
+  PropType,
+} from 'vue';
+import type { Protocol } from '@/types';
 import { truncateAddress } from '@/utils';
-import { AeScan } from '@/protocols/aeternity/libs/AeScan';
-import { useAeNetworkSettings } from '@/protocols/aeternity/composables';
+import { PROTOCOL_AETERNITY } from '@/constants';
 
 import ExternalLinkIcon from '@/icons/external-link.svg?vue-component';
+import { ProtocolAdapterFactory } from '@/lib/ProtocolAdapterFactory';
```
Comment (peronczyk) [inline src/popup/components/AddressTruncated.vue:35]: Please move this line above the icons and components.

File: src/protocols/bitcoin/libs/blockstream.ts
Comment (peronczyk) [inline src/protocols/bitcoin/libs/blockstream.ts:1]: Please use capital letter for the file name.

**Review by peronczyk — APPROVED** [review]

File: src/protocols/bitcoin/composables/btcNetworkSettings.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    () => activeNetwork.value.protocols[PROTOCOL_BITCOIN] as any, // TODO - type btc network
+  );
+
+  const btcActiveNetworkPredefinedSettings = computed(
+    (): any => { // TODO - type IBtcNetworkPredefinedSettings network
+      const networkType: NetworkTypeDefault = (activeNetwork.value.type === NETWORK_TYPE_CUSTOM)
+        ? NETWORK_TYPE_TESTNET
+        : activeNetwork.value.type;
+      const additionalSettings = BTC_NETWORK_ADDITIONAL_SETTINGS[networkType];
+      return additionalSettings || {};
```
Comment (konradodo) [inline src/protocols/bitcoin/composables/btcNetworkSettings.ts:24]:
```suggestion
      return BTC_NETWORK_ADDITIONAL_SETTINGS[networkType] || {};
```

**Review by konradodo — APPROVED** [review]

**Review by CedrikNikita — APPROVED** [review]
Code changes look reasonable.

**Comment by Liubov-crypto** [comment]
LGTM

File: src/popup/components/AccountSelector.vue [hunk 3]
```diff
@@ -27,6 +27,7 @@
         </FormSelect>
       </BtnPill>
       <AddressTruncated
+        :protocol="modelValue.protocol"
```
Comment (martinkaintas) [inline src/popup/components/AccountSelector.vue:30]: `modelValue` type should be updated as it is `[String, Number]` right now

**Pull request merged**